onedal/common/_policy.py

```diff
@@ -34,6 +34,10 @@ def _get_policy(queue, *data):
     return _HostInteropPolicy()
 
 
+def _get_spmd_policy(queue):
+    return _SPMDDataParallelInteropPolicy(queue)
```

Comment:
Undefined _SPMDDataParallelInteropPolicy if not _is_dpc_backend

Reply:
I guess it is not needed in such case?

Reply:
Warning is required to explain default policy for user